buildpipeline/Core-Setup-Windows-BT.json

```diff
@@ -78,6 +133,141 @@
       "enabled": true,
       "continueOnError": false,
       "alwaysRun": false,
+      "displayName": "Build binaries",
```

Comment:
We were, but we were doing it by just running build.cmd as a monolith - https://github.com/dotnet/core-setup/blob/master/buildpipeline/Core-Setup-Windows-BT.json#L60-L76. I'm copying the pattern from the signing job for this PR, where we run build.cmd to generate version assets, then explicitly build src.builds to build binaries.